src/eth1spec/evm/error.py

```diff
@@ -0,0 +1,29 @@
+"""
+EVM Runtime Errors
+---
```

Comment:
```suggestion
EVM Runtime Errors
------------------
```

Underlines have to match their headings (I think.)